src/main.js

```diff
@@ -570,18 +570,14 @@ window.LinkedinToResumeJson = (() => {
         });
     };
 
-    LinkedinToResumeJson.prototype.parseViaInternalApi = async function parseViaInternalApi() {
+    LinkedinToResumeJson.prototype.parseViaInternalApiFullProfile = async function parseViaInternalApiFullProfile() {
         try {
-            let apiSuccessCount = 0;
-            let fullProfileEndpointSuccess = false;
-
             // Get full profile
             const fullProfileView = await this.voyagerFetch(_voyagerEndpoints.fullProfileView);
             if (fullProfileView && typeof fullProfileView.data === 'object') {
                 // Try to use the same parser that I use for embedded
                 const profileParserResult = parseProfileSchemaJSON(this, fullProfileView);
                 if (profileParserResult) {
-                    apiSuccessCount++;
                     fullProfileEndpointSuccess = true;
```

Comment:
This variable (`fullProfileEndpointSuccess`) is actually out of scope now. This doesn't throw an error, but it pollutes the global environment and is unused here anyways.